fontforge/stemdb.c

```diff
@@ -2362,8 +2362,7 @@ return( 0 );
     /* fonts or fonts with quadratic splines). */
     /* But do that only for colinear spline segments and ensure that there are  */
     /* no bends between two splines. */
-    if ( !tp && ( !fp || t > 0.5 ) &&
-	topd->colinear && &other->to->next != NULL ) {
```

Comment:
Change it to `topd->next != NULL` rather than deleting it. That's the likely original intention and it can't hurt.

Reply:
Applied